scio-test/src/test/scala/com/spotify/scio/testing/JobTestTest.scala

```diff
@@ -21,12 +21,11 @@ import com.google.common.collect.ImmutableMap
 import com.google.datastore.v1.Entity
 import com.google.datastore.v1.client.DatastoreHelper.{makeKey, makeValue}
 import com.spotify.scio._
-import com.spotify.scio.coders.Coder
 import com.spotify.scio.avro.AvroUtils.{newGenericRecord, newSpecificRecord}
-import com.spotify.scio.avro._
+import com.spotify.scio.avro.{AvroUtils, _}
```

Comment:
`AvroUtils.newGenericRecord` is alread in imported in L:24

Reply:
Will fix this.